templates/download/intel-iei-tank-870.html

```diff
@@ -1,83 +1,70 @@
-            <li>Download and flash an <a href="/download/desktop">Ubuntu Desktop {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr></a> image on the first USB flash drive by following the live USB Ubuntu Desktop tutorial for <a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>, <a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>, or  <a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a>.</li>
-            <li>Copy the Ubuntu Core image file to the second USB flash drive.</li>
-          </ol>
-        </div>
-      </li>
-      <li class="p-stepped-list__item">
-        <h3 class="p-stepped-list__title">
-
-          Boot the live Ubuntu Desktop image
-        </h3>
-        <div class="p-stepped-list__content">
-          <ol class="u-no-margin--left">
-            <li>Connect the keyboard and monitor to the Intel IEI TANK 870.</li>
-            <li>Insert the first USB flash drive, containing Ubuntu Desktop {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr>.</li>
-            <li>Start the device and press F7 to enter the boot menu.</li>
-            <li>Select the USB flash drive as a boot option.</li>
-            <li>Select "Try Ubuntu without installing".</li>
-          </ol>
-        </div>
-      </li>
-      <li class="p-stepped-list__item">
-        <h3 class="p-stepped-list__title">
-
-            Flash Ubuntu Core to the internal memory
-          </h3>
+          <hr class="p-rule u-hide--large" />
+          <h2 class="p-stepped-list__title">Download Ubuntu Core</h2>
+          <div class="p-stepped-list__content">
+            <p>Download the <a href="https://cdimage.ubuntu.com/ubuntu-core/18/stable/current/ubuntu-core-18-amd64+kassel.img.xz">Ubuntu Core 18 image for the Intel IEI TANK 870</a>.</p>
+            <p>You can then verify the integrity of the download using the associated <a href="https://cdimage.ubuntu.com/ubuntu-core/18/stable/current/SHA256SUMS">SHA256SUMS file</a> and the <code>md5sum</code> command on most Linux distributions.</p>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <hr class="p-rule u-hide--large" />
+          <h2 class="p-stepped-list__title">Prepare the two USB flash drives</h2>
+          <div class="p-stepped-list__content">
+            <ol class="u-no-margin--left">
+              <li>Download and flash an <a href="/download/desktop">Ubuntu Desktop {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr></a> image on the first USB flash drive by following the live USB Ubuntu Desktop tutorial for <a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>, <a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>, or  <a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a>.</li>
+              <li>Copy the Ubuntu Core image file to the second USB flash drive.</li>
+            </ol>
```

Comment:
Shouldn't these have the separator? i.e. `p-list--divided` classes? but up to @lyubomir-popov

Reply:
Oh I think you're right, numbered lists on the design do have that separator. He already left design +1 but I'll ping him to double check